src/main/java/frc/robot/competition/Robot.java

```diff
@@ -85,13 +86,18 @@ public void robotPeriodic() {
   }
 
   @Override
-  public void disabledInit() {}
+  public void disabledInit() {
+    m_robotContainer.m_chassis.setBrake(false);
+  }
 
   @Override
   public void disabledPeriodic() {}
 
   @Override
-  public void disabledExit() {}
+  public void disabledExit() {
+    m_robotContainer.m_chassis.setBrake(
+        RobotConstants.MODULE_CONFIG.driveIdleMode == IdleMode.kBrake ? true : false);
```

Comment:
```suggestion
        RobotConstants.MODULE_CONFIG.driveIdleMode == IdleMode.kBrake);
```

Alternatively, change `setBrake` to `setIdle` and just have it take in an `IdleMode` object